llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -474,8 +473,8 @@ class FilterChooser {
   //
   // Returns false if there exists any uninitialized bit value in the range.
```

Comment:
Please update comment. It no longer matches the behavior.

Reply:
Done.